guide/how-it-works/liquidity.md

```diff
@@ -306,6 +306,49 @@ Consider how you can help the user with channel management without them even kno
 
 By combining the business incentives of an LSP, clever engineering, and good design, you can build a bitcoin product that makes using the lightning network very easy for the user.
 
+## Channel reserve
+
+### What is a channel reserve?
+
+A channel reserve is an amount that is set aside by each channel participant which ensures neither have 'nothing at stake' if a cheating attempt occurs. This reserve can not be spent, and is held aside for the entirety of the channels lifetime.
+
+Channel reserves make cheating attempts less economical. When one channel party attempts to cheat the other and they are caught, a [penalty transaction](https://fiatjaf.com/73095980.html) can be used to steal all the cheating parties bitcoin as punishment. Channel reserves makes it so cheating attempts always have something at stake making this less likely to occur.
+
+### What is the reserve amount?
```

Comment:
I think this section would really benefit from some graphics depicting channel reserve in the same style as the ones above. The graphics would help explain a complicated topic and also help the page look nice from top to bottom.
@mouxdesign If you'd like, I could make some graphics and send you the Figma link or commit to your branch.

Reply:
Fully agree with you @sbddesign , feel free to design a graphic.

Reply:
Ok, I just pushed some graphics.